lib/instrumentation/modules/cassandra-driver.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+ * TODO: Support batch queries
```

Comment:
Do you want to release without support for batch queries?

Reply:
Looking further down it seems like support for batch queries have been added. Maybe just old comment?

Reply:
Yep. Old comment...removing...